CHANGELOG.md

```diff
@@ -3,8 +3,8 @@
 ## 2.2.0 - 2024-01-03
 
 ### Changed
-- Get issuer and vc holder used in test data generation from test
-  config. This will allow testers to specify the issuer and/or vc holder for
+- Get issuer and VC holder used in test data generation from test
+  config. This will allow testers to specify the issuer and/or VC holder for
```

Comment:
```suggestion
- Get VC issuer and holder from test config for use in test data generation.
  This will allow testers to specify the VC issuer and/or holder for
```